main.go

```diff
@@ -35,6 +36,11 @@ import (
 	"sync"
 	"time"
 
+	"github.com/joho/godotenv"
+
+	"github.com/percona/pmm-managed/services/config"
+	"github.com/percona/pmm-managed/services/telemetry"
```

Comment:
please fix imports, now we have 5 blocks

Reply:
it should be fixed with tools, right? `make format` doesn't change import